test/web3.eth.getSyncing.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+var chai = require('chai');
+var Web3 = require('../index');
```

Comment:
Also, make sure to use strict mode in every file you rewrite. I'v noticed multiple swallowed errors because of missing strict mode.